content/integrate/redis-data-integration/ingest/reference/cli/redis-di-add-context.md

```diff
@@ -0,0 +1,145 @@
+---
+Title: redis-di add-context
+linkTitle: redis-di add-context
+description: Adds a new context 
+weight: 10
+alwaysopen: false
+categories: ["redis-di"]
+aliases:
+---
+
+
+Adds a new context
```

Comment:
This description gets added to the page automatically when it is published, so having it here makes it appear twice. This already happens with the existing docs, btw (eg, https://redis.io/docs/latest/integrate/redis-data-integration/reference/cli/redis-di/), but it would be nice if we could fix it. Also, the current CLI reference folder has an _index.md page, which means the commands get put in a CLI category in the sidebar. This is easy enough to add manually before release but if it can be automated then that's probably better.

Reply:
@spicy-sauce can we do these moving forward? I think for this release we do manually